packages/api/test/e2e/rx-queries.spec.ts

```diff
@@ -29,10 +32,8 @@ describe.skip('e2e queries', () => {
 
   it('queries state for a balance', (done) => {
     api.query.balances.freeBalance(keyring.alice.address()).subscribe((balance) => {
-      expect(
-        balance.isZero()
-      ).toBe(false);
 
+      expect(balance).toBeInstanceOf(BN);
```

Comment:
Same: we should test isZero false here

Reply:
Doesn't work for rx, though, here I'm still getting the `Property 'isZero' does not exist on type 'Codec'.` tslint error....

Reply:
This is because we right now don't have exact types in the `api.query` query functions, so they only return Codec. We need to manually cast.

`(balance as Balance).isZero()` should work

Reply:
thanks, solved!